### PR TITLE
User avatar

### PR DESCRIPTION
**Pull request by derpdead**
# Description

Add support for user avatar

Issue # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Unit Tests
- [ ] e2e Test

# Checklist:

- [x] I have read the contribution requirements and fulfil them.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by dawidstrix** [comment]
Build 131 Fixed

File: modules/@ergonode/users/src/components/Forms/UserAvatarForm.vue [hunk 1]
```diff
@@ -50,6 +54,7 @@ export default {
             '__setState',
         ]),
         uploadValue(value = '') {
+            console.log(value);
```
Comment (bleto) [inline modules/@ergonode/users/src/components/Forms/UserAvatarForm.vue:57]: to remove

File: modules/@ergonode/users/src/components/Inputs/UploadAvatar.vue [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+            if (file) {
+                const {
+                    name,
+                } = file;
+
+                const formData = new FormData();
+                formData.append('upload', file, name);
+
+                this.$axios.$post(`${this.languageCode}/accounts/${this.userId}/avatar`, formData).then(() => {
```
Comment (bleto) [inline modules/@ergonode/users/src/components/Inputs/UploadAvatar.vue:102]: move to service ?

File: modules/@ergonode/users/src/store/users/state.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ export default () => ({
     firstName: '',
     lastName: '',
     language: '',
+    languageCode: '',
```
Comment (bleto) [inline modules/@ergonode/users/src/store/users/state.js:12]: what is different between language and languageCode ? We need this ?

**Review by bleto — DISMISSED** [review]

**Review by bleto — APPROVED** [review]

**Pull request merged**